notebooks/helpers.py

```diff
@@ -61,12 +61,11 @@ def load_titanic(test_size=.25, feature_skip_tuple=(), random_state=1999):
         strings = {k: line_dict[k] for k in string_keys}
         numeric_labels[n] = line_dict["survived"]
 
-    sss = StratifiedShuffleSplit(numeric_labels, n_iter=1, test_size=test_size,
-                                 random_state=12)
+    sss = StratifiedShuffleSplit(n_iter=1, test_size=test_size, random_state=12)
     # This is a weird way to get the indices but it works
     train_idx = None
     test_idx = None
-    for train_idx, test_idx in sss:
+    for train_idx, test_idx in sss.split(numeric_data, numeric_labels):
```

Comment:
That's the hacky code that was already there. I just updated the API calls to sklearn 0.18.